WindowsServerDocs/remote/remote-desktop-services/rds-client-access-license.md

```diff
@@ -18,11 +18,11 @@ manager: dongill
 
 >Applies to: Windows Server (Semi-Annual Channel), Windows Server 2019, Windows Server 2016
 
-Each user and device that connects to a Remote Desktop Session host needs a client access licenses (CAL). You use RD Licensing to install, issue, and track RDS CALs.  
+Each user and device that connects to a Remote Desktop Session host needs a client access license (CAL). You use RD Licensing to install, issue, and track RDS CALs.  
 
-When a user or a device connects to an RD Session Host server, the RD Session Host server determines if an RDS CAL is needed. The RD Session Host server then requests an RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
+When a user or a device connects to an RD Session Host server, the RD Session Host server determines if a RDS CAL is needed. The RD Session Host server then requests a RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
```

Comment:
```suggestion
When a user or a device connects to an RD Session Host server, the RD Session Host server determines if an RDS CAL is needed. The RD Session Host server then requests an RDS CAL from the Remote Desktop license server. If an appropriate RDS CAL is available from a license server, the RDS CAL is issued to the client, and the client is able to connect to the RD Session Host server and from there to the desktop or apps they're trying to use.
```

Reply:
From the PR author:
> "a Remote Desktop Server" <-- Acronyms are treated based on the first word not letter.

(see also my comment https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issuecomment-590071034 and the main PR comment  https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issue-378668444 )

Reply:
(@PhilipElder : Feel free to mark this conversation as Resolved)

Reply:
Mind you, from my own experience, English speech usually adapts to how acronyms are pronounced.
https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html
https://www.grammar.com/a-vs-an-when-to-use/

Reply:
Neat, so I sit corrected as I dig around:
https://ontariotraining.net/grammar-tip-a-or-an-with-acronyms/
https://www.grammar.com/a-vs-an-when-to-use/

Things change. :)

Reply:
😄 Fair enough, glad it was a light-hearted debate (tempered debates tend to make me feel less helpful & useful).
I presume from your reply that we can treat "RDS" as an _initialism_ in this case. :)
Looking forward to your merge (commit) of that change. BTW, you may want to edit your PR comment too.

Reply:
I'm just learning GitHub and its language/semantics.

Which "PR" comment do I need to edit? The one with the a/an RDS?

Reply:
Oh. Right. I should have been more precise. Your initial PR opening statement:

https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946 (top of page) | https://github.com/MicrosoftDocs/windowsserverdocs/pull/3946#issue-378668444 (1st comment)
> It's " ... needs a client access license (CAL)."
> "a Remote Desktop Server" <-- Acronyms are treated based on the first _word_ not letter.
> A couple more tweaks.
> Added the Per Device and Per User examples for RDS CALs.